analyzer/javacg-opal/src/main/java/eu/fasten/analyzer/javacgopal/data/OPALPartialCallGraphConstructor.java

```diff
@@ -127,12 +126,39 @@ public static PartialJavaCallGraph createPartialJavaCG(
                     partialCallGraph.graph);
         } finally {
             if (file != null) {
-            	// TODO use apache commons FileUtils instead
+                // TODO use apache commons FileUtils instead
                 file.delete();
             }
         }
     }
 
+    /**
+     * Creates RevisionCallGraph from a local JAR given a Maven coordinate.
+     *
+     * @param coordinate maven coordinate of the revision to be processed
+     * @param timestamp  timestamp of the revision release
+     * @return RevisionCallGraph of the given coordinate.
+     * @throws FileNotFoundException
+     */
+    public static PartialJavaCallGraph createPCGFromLocalJar(final MavenCoordinate coordinate,
```

Comment:
Please merge the two `createPCG` methods into one. There is NO need for a case distinction, both should do exactly the same...